roles/load-balance/files/load-balancer.conf

```diff
@@ -0,0 +1,49 @@
+###############################################################################
+#                                                                             #
+#           THIS FILE IS MANAGED BY ANSIBLE! CHANGES ARE OVERWRITTEN.         #
+#            https://github.com/jwflory/rit-nssa-320-infrastructure           #
+#                                                                             #
+###############################################################################
+
+# Official English Documentation: http://nginx.org/en/docs/
+
+http {
+	upstream backend {
+      least_conn;
+      server 10.0.2.1; 
+      server 10.0.2.2;
+      server 10.0.2.3;
+
+      # not sure if this will work
+      sticky learn
+       create=$upstream_cookie_examplecookie
+       lookup=$cookie_examplecookie
+       zone=client_sessions:1m
+       timeout=1h
+       sync;
+	}
+
+	# This server accepts all traffic to port 80 and passes it to the upstream. 
+
+	server {
+		listen 80; 
+
+		location / {
+			proxy_pass http://backend;
+		}
+	}
+}
+
+# I dont remember if we ever set up https but if we did use this
+server {
+   listen 443 ssl;
+   server_name explorecalifornia.com www.explorecalifornia.com;
+
+   ssl_certificate /etc/letsencrypt/live/domain_name/cert.pem;
+   ssl_certificate_key /etc/letsencrypt/live/domain_name/privkey.pem;
+   ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+
+   location / {
+      proxy_pass http://backend;
+   }
+}
```

Comment:
Did you have a reason to exclude this block from the `http { }` block above?

Reply:
Just not sure if it works with our set up, but it would likely need to be in that block